timesheet_automatic_remind/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Timesheet Automatic Remind
+==========================
+
+
+Develop the automatic emails to send every morning to people who
```

Comment:
I made some modifications in the README in previous version. Can you retrieve them?